client/payments/data/action-types.ts

```diff
@@ -0,0 +1,6 @@
+export enum ACTION_TYPES {
+	GET_PAYMENT_GATEWAYS_SUCCESS = 'GET_PAYMENT_GATEWAYS_SUCCESS',
+	UPDATE_PAYMENT_GATEWAY_SUCCESS = 'UPDATE_PAYMENT_GATEWAY_SUCCESS',
+	SET_ERROR = 'SET_ERROR',
+	SET_IS_REQUESTING = 'SET_IS_REQUESTING',
```

Comment:
With `_SUCCESS` and `_ERROR` we won't need this one anymore.  Whenever those actions occur, we can set `isRequesting` to `true` in the reducer.